### ISSUE TITLE
Connect method for ActiveRecord Database Connection

### ISSUE THREAD

**Issue by ananyo2012** [SciRuby/daru#316]
We can have a connect method for establishing ActiveRecord connections for use in `from_activerecord` method by simply passing the db name, username and password as arguments. As of now we have to establish database connection manually.
```ruby
ActiveRecord::Base.establish_connection(
  adapter:  'mysql2',
  host:     'localhost',
  username: 'me',
  password: 'secret',
  database: 'activerecord'
)
```

**Comment by v0dro** [comment]
@mrkn WDYT?

**Comment by mrkn** [comment]
@ananyo2012 I want to see a code example of your suggestion.

**Comment by zverok** [comment]
I believe the main use case for `from_activerecord` is "I already have a connection" (from inside Rails app, or something). But I'm open to use cases/code examples for this feature.

**Comment by ananyo2012** [comment]
In `lib/daru/IO/io.rb` we can have
```ruby
module IO
  class << self
    def activerecord_conn(adapter, user, password, db_name)
      ActiveRecord::Base.establish_connection(
        adapter:  adapter,
        host:     'localhost',
        username: user,
        password: password,
        database: db_name
      )
    end
  end
end
````
This can be called before calling the `Daru::DataFrame.from_activerecord` method to establish connection.

**Comment by v0dro** [comment]
@mrkn ping!

**Comment by mrkn** [comment]
I don't think your `activerecord_conn` method is not useful for AR users.

The parameters of AR::Base.establish_connection is not only the combination of adapter, user, password, and db_name, but it also accepts other optional parameters such as DBMS-specific connection parameters.

Also this method can be called with one `url` parameter, the example value is: `mysql2://db_user:db_pass@localhost:3306/db_name`.

**Comment by ananyo2012** [comment]
@mrkn Yes I guess the different formats in the AR connect method gives it enough flexibility as per the database and environments. But for a first timer approaching the codebase it is pretty hard to know how it works around. It would be great if we can simplify things for them. Can we improve the documentation for this a bit or atleast giving the link you have pointed out in the docs?
